include/mbedtls/x509_crt.h

```diff
@@ -598,7 +601,7 @@ int mbedtls_x509_crt_parse_path(mbedtls_x509_crt *chain, const char *path);
  * \param san      The target structure to populate with the parsed presentation
  *                 of the subject alternative name encoded in \p san_raw.
  *
- * \note           Only "dnsName" and "otherName" of type hardware_module_name
+ * \note           Only "dnsName" and "otherName" and "rfc822Name" of type hardware_module_name
```

Comment:
This description is wrong. "hardware_module_name" is a subtype of "otherName", and "rfc822Name" is on the same level as "otherName". Also, while we're at it, I believe that the next line should mention RFC 5280, not 4180.